Multisig/UI/Transactions/TransactionDetails/TransactionConfirmationsView/TransactionConfirmationsView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ConfirmationsView.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/10/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionConfirmationsView: View {
+    let transaction: BaseTransactionViewModel
+    let safe: Safe
+
+    var body: some View {
+        return VStack(alignment: .leading, spacing: 1) {
+            TransactionConfirmationCell(address: transaction.confirmations!.first!.address, style: .created)
```

Comment:
please add an assert before that somewhere in the initializer or in the body, something like:

```
assert(tranaction.confirmations?.count > 1)
```